[2024-09-28T11:53:30.994+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-09-28T11:53:31.051+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: csv_to_postgres_dag.generate_insert_queries manual__2024-09-28T11:53:28.201290+00:00 [queued]>
[2024-09-28T11:53:31.072+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: csv_to_postgres_dag.generate_insert_queries manual__2024-09-28T11:53:28.201290+00:00 [queued]>
[2024-09-28T11:53:31.074+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 1
[2024-09-28T11:53:31.107+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): generate_insert_queries> on 2024-09-28 11:53:28.201290+00:00
[2024-09-28T11:53:31.121+0000] {standard_task_runner.py:63} INFO - Started process 3347 to run task
[2024-09-28T11:53:31.127+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'csv_to_postgres_dag', 'generate_insert_queries', 'manual__2024-09-28T11:53:28.201290+00:00', '--job-id', '16', '--raw', '--subdir', 'DAGS_FOLDER/job.py', '--cfg-path', '/tmp/tmpi8agwe76']
[2024-09-28T11:53:31.129+0000] {standard_task_runner.py:91} INFO - Job 16: Subtask generate_insert_queries
[2024-09-28T11:53:31.206+0000] {task_command.py:426} INFO - Running <TaskInstance: csv_to_postgres_dag.generate_insert_queries manual__2024-09-28T11:53:28.201290+00:00 [running]> on host c39a3d3a6c9a
[2024-09-28T11:53:31.343+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='csv_to_postgres_dag' AIRFLOW_CTX_TASK_ID='generate_insert_queries' AIRFLOW_CTX_EXECUTION_DATE='2024-09-28T11:53:28.201290+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-09-28T11:53:28.201290+00:00'
[2024-09-28T11:53:31.344+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-09-28T11:53:31.377+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-09-28T11:53:31.380+0000] {taskinstance.py:2905} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.9/site-packages/airflow/models/taskinstance.py", line 465, in _execute_task
    result = _execute_callable(context=context, **execute_callable_kwargs)
  File "/home/airflow/.local/lib/python3.9/site-packages/airflow/models/taskinstance.py", line 432, in _execute_callable
    return execute_callable(context=context, **execute_callable_kwargs)
  File "/home/airflow/.local/lib/python3.9/site-packages/airflow/models/baseoperator.py", line 400, in wrapper
    return func(self, *args, **kwargs)
  File "/home/airflow/.local/lib/python3.9/site-packages/airflow/operators/python.py", line 235, in execute
    return_value = self.execute_callable()
  File "/home/airflow/.local/lib/python3.9/site-packages/airflow/operators/python.py", line 252, in execute_callable
    return self.python_callable(*self.op_args, **self.op_kwargs)
  File "/opt/airflow/dags/job.py", line 25, in generate_insert_queries
    with open('sql/insert_queries.sql', 'w') as f:
FileNotFoundError: [Errno 2] No such file or directory: 'sql/insert_queries.sql'
[2024-09-28T11:53:31.402+0000] {taskinstance.py:1206} INFO - Marking task as FAILED. dag_id=csv_to_postgres_dag, task_id=generate_insert_queries, run_id=manual__2024-09-28T11:53:28.201290+00:00, execution_date=20240928T115328, start_date=20240928T115331, end_date=20240928T115331
[2024-09-28T11:53:31.420+0000] {standard_task_runner.py:110} ERROR - Failed to execute job 16 for task generate_insert_queries ([Errno 2] No such file or directory: 'sql/insert_queries.sql'; 3347)
[2024-09-28T11:53:31.460+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 1
[2024-09-28T11:53:31.480+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.9/site-packages/***/models/baseoperator.py:1296 AirflowProviderDeprecationWarning: Call to deprecated class PostgresOperator. (Please use `***.providers.common.sql.operators.sql.SQLExecuteQueryOperator`.Also, you can provide `hook_params={'schema': <database>}`.)
[2024-09-28T11:53:31.499+0000] {taskinstance.py:3498} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-09-28T11:53:31.503+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
